Backpack/src/main/java/net/skyscanner/backpack/calendar2/data/CalendarDispatchers.kt

```diff
@@ -36,10 +35,12 @@ object CalendarDispatchers {
   val Background: CoroutineDispatcher
     get() = background
 
+  @VisibleForTesting
```

Comment:
only these two methods are restricted for testing - the others can be used validly in production code